packages/flutter/lib/src/material/search.dart

```diff
@@ -233,6 +233,7 @@ abstract class SearchDelegate<T> {
   Animation<double> get transitionAnimation => _proxyAnimation;
 
   final FocusNode _focusNode = FocusNode();
```

Comment:
it's kinda sketchy that this class is creating a focus node but not disposing of it.
(Wasn't an issue before because they didn't need to be disposed.)

cc @goderbauer